lib/yoti/share/issuance_details.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'base64'
+
+module Yoti
+  module Share
+    class AttributeIssuanceDetails
+      attr_reader :token
+      attr_reader :attributes
+      attr_reader :expiry_date
+
+      #
+      # Constructor
+      #
+      # @param [Yoti::Protobuf::Sharepubapi::ThirdPartyAttribute] data_entry
+      #
+      def initialize(data_entry)
+        @token = Base64.encode64(data_entry.issuance_token)
+        begin
+          @expiry_date = DateTime.parse(data_entry.issuing_attributes.expiry_date)
+        rescue ArgumentError
+          @expiry_date = nil
+        end
+        @attributes = data_entry.issuing_attributes.definitions
```

Comment:
Should this be a `Array<Yoti::Share::AttributeDefinition>` instead of `Google::Protobuf::RepeatedField`?